src/SFA.DAS.AssessorService.Web/StartupConfiguration/AuthenticationStartup.cs

```diff
@@ -218,33 +230,32 @@ public static class AuthenticationStartup
                                  {
                                      identity.AddClaim(new Claim("UserId", user?.Id.ToString()));
                                      identity.AddClaim(new Claim(
-                                        "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/upn",
-                                        user?.Username));
+                                         "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/upn",
+                                         user?.Username));
                                      if (user.EndPointAssessorOrganisationId != null)
                                      {
                                          var organisation =
-                                            await orgClient.GetEpaOrganisation(user.EndPointAssessorOrganisationId);
+                                             await orgClient.GetEpaOrganisation(user.EndPointAssessorOrganisationId);
 
-                                         if (organisation.ApiEnabled && !string.IsNullOrEmpty(organisation.ApiUser))
-                                         {
-                                             identity.AddClaim(new Claim("http://schemas.portal.com/service",
-                                                Roles.ExternalApiAccess));
-                                             identity.AddClaim(new Claim("http://schemas.portal.com/service",
-                                                Roles.EpaoUser));
-                                         }
+                                         //Fix for ON-1759 , for now untill roles and privileges are applied properly 
+                                         //allow everyone to see ExternalApiAccess
+                                         identity.AddClaim(new Claim("http://schemas.portal.com/service",
+                                             Roles.ExternalApiAccess));
+                                         identity.AddClaim(new Claim("http://schemas.portal.com/service",
+                                             Roles.EpaoUser));
 
                                          identity.AddClaim(new Claim("http://schemas.portal.com/ukprn",
-                                            organisation?.Ukprn == null ? "" : organisation?.Ukprn.ToString()));
+                                             organisation?.Ukprn == null ? "" : organisation?.Ukprn.ToString()));
 
-                                         var orgName = organisation.OrganisationData?.LegalName ??
-                                                       organisation.OrganisationData?.TradingName ??
-                                                       organisation.Name;
+                                         var orgName = organisation?.OrganisationData?.LegalName ??
+                                                       organisation?.OrganisationData?.TradingName ??
+                                                       organisation?.Name;
```

Comment:
From @alandburns 

> How is it possible to have No organisation?
> And if there is no value in the orgName claim this will cause an Unhandled Exception.

Reply:
fixed this with a null check futher up